### PR TITLE
Remove json schema parameter

### PR DESCRIPTION
**Pull request by avida**
source-acceptance-test framework not longer required json_schema parameters
   from catalog file. This parameter is verbose, makes reading config file
   complicated and could be misleading when debugging acceptance test issues.

## What
Removes json_schema parameter for all configured_catalog_file from acceptance-test-config.yml config since we no longer need it.

## Recommended reading order
1. `x.json`

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Secrets are annotated with `airbyte_secret` in the connector's spec
- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [ ] Unit & integration tests added as appropriate (and are passing)
    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] `README.md`
    - [ ] `docs/SUMMARY.md` if it's a new connector
    - [ ] Created or updated reference docs in `docs/integrations/<source or destination>/<name>`.
    - [ ] Changelog in the appropriate page in `docs/integrations/...`. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md` contains a reference to the new connector
    - [ ] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [ ] Build is successful
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Follow up tickets have been created
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

<details><summary> <strong> Connector Generator checklist </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

**Review by htrueman — COMMENTED** [review]
Overall LGTM. Please make sure to format all files with `./gradlew format` and check why do tests fail on the most connectors.

**Comment by keu** [comment]
@avida in order to move this ASAP, let's do the following:
- remove anything not related to json_schema removal (dummy tests, etc)
- resolve conflicts with stripe
- format json

**Comment by keu** [comment]
no need to run tests on connectors, we will to publish new versions, we will check tests after night build

**Review by Zirochkaa — APPROVED** [review]

**Review by yevhenii-ldv — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/configured_catalog.json [hunk 1]
```diff
@@ -1,56 +0,0 @@
...
-            }
-          }
-        },
-        "supported_sync_modes": ["full_refresh", "incremental"]
-      },
-      "sync_mode": "incremental",
-      "destination_sync_mode": "append"
-    }
-  ]
-}
```
Comment (tuliren) [inline airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/configured_catalog.json:56]: This file is completely empty now, which breaks the spotless check. The non-schema part should be reserved in this file, right? @avida

**Pull request merged**